### PR TITLE
Resolved #296

### LINKED ISSUE #296: Gitea returns a 500 when searching for %00
- Gitea version (or commit ref): 27d66855eb642b22852b557cbb8ffd7781d73d4e
- Git version: 2.10.2
- Operating system: Arch linux
- Database:
  - [x] PostgreSQL 9.6.1 en_US.utf8
  - [ ] MySQL
  - [ ] SQLite
- Log gist: `2016/11/28 23:43:28 [...itea/routers/home.go:93 RenderRepoSearch()] [E] SearchRepositoryByName: Count: pq: invalid byte sequence for encoding "UTF8": 0x00`

Go to your local Gitea server, enter %00 behind ?q=  (like http://localhost/explore/repos?q=%00) and Gitea returns a 500 error

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: templates/explore/organizations.tmpl [hunk 1]
```diff
@@ -24,6 +24,8 @@
 						    </div>
 						  </div>
 						</div>
+					{{else}}
+					<div>{{$.i18n.Tr "explore.org_no_results"}}</div>
```
Comment (tboerger) [inline templates/explore/organizations.tmpl]: Indentation

File: templates/explore/repo_list.tmpl [hunk 2]
```diff
@@ -19,5 +19,9 @@
 			{{if .DescriptionHTML}}<p class="has-emoji">{{.DescriptionHTML}}</p>{{end}}
 			<p class="time">{{$.i18n.Tr "org.repo_updated"}} {{TimeSince .Updated $.i18n.Lang}}</p>
 		</div>
+	{{else}}
+	<div>
```
Comment (tboerger) [inline templates/explore/repo_list.tmpl:23]: Indentation

File: templates/explore/users.tmpl [hunk 3]
```diff
@@ -24,6 +24,8 @@
 						    </div>
 						  </div>
 						</div>
+					{{else}}
+					<div>{{$.i18n.Tr "explore.user_no_results"}}</div>
```
Comment (tboerger) [inline templates/explore/users.tmpl]: Indentation

**Review by tboerger — COMMENTED** [review]
Some nitpicking about the template indentation, otherwise LGTM

**Comment by bkcsoft** [comment]
LGTM

**Comment by tboerger** [comment]
I will still give you my LGTM

**Comment by strk** [comment]
Please make PR titles meaningful, so those of us receiving notifications by mail can tell what it is about w/out having to open a referenced ticket to know. (and I think it'll also help with automating Changelog if we take the GitLab path)

**Pull request merged**